src/pybind/mgr/cephadm/module.py

```diff
@@ -1239,6 +1263,10 @@ def _add_host(self, spec):
         :param host: host name
         """
         assert_valid_host(spec.hostname)
+
+        # Copy cephadm used in this cluster to the remote host
+        self._copy_cephadm(spec.hostname, spec.addr)
```

Comment:
this is too brittle. We have to make sure we create the cephadm binary whenever it is missing